activiti-cloud-starter-runtime-bundle/src/test/java/org/activiti/cloud/starter/tests/runtime/ProcessInstanceIT.java

```diff
@@ -120,7 +120,6 @@ public void shouldStartProcess() {
         assertThat(returnedProcInst.getInitiator()).isEqualTo(keycloakTestUser);//will only match if using username not id
         assertThat(returnedProcInst.getBusinessKey()).isEqualTo("business_key");
         assertThat(returnedProcInst.getAppName()).isEqualTo(runtimeBundleProperties.getAppName());
-        assertThat(returnedProcInst.getAppVersion()).isEqualTo(runtimeBundleProperties.getAppVersion());
```

Comment:
This should not be removed, only change the value to the default application version value.

Reply:
The thing is that the AppVersion is not coming from runtimeBundleProperties anymore.
That is also a problem for the RUNTIME_BUNDLE_INFO_HEADERS in AuditProducerIT.
The APP_VERSION header is set in ExecutionContextMessageBuilderAppender class, and not in the RuntimeBundleInfoMessageBuilderAppender.
We should discuss this

Reply:
Ok, let's handle the missing headers in a separated PR.